examples/test_sparql.metta

```diff
@@ -0,0 +1,74 @@
+!(extend-py! motto.sparql_extension.sparql_extension)
```

Comment:
As a minor remark, if `sparql_extension` is implemented as a Python module with proper `__init__.py`, then it would not be necessary to put additional `.sparql_extension`. I'd also renamed `sparql_extension` to a shorter version. Maybe, just `sparql` or `sparql_gate` or `sparqlme` or `sparql_ext` or whatever else.